source/sonata_report.rst

```diff
@@ -105,49 +105,33 @@ This JSON file defines named sets of compartment targets. Each set specifies the
       "population": "S1nonbarrel_neurons",
       "compartment_set": [
         [0, 1, 0.1],
-        [3, 5, 0.7],
         [0, 2, 0.3]
+        [3, 5, 0.7],
       ]
     }
   }
 
-*   ``population``: The name of the node population of the ``node_id``\s in ``compartment_set``.
-*   ``compartment_set``: A list of lists, where each list is ``[node_id, section_id, offset]``. Each of the ``[node_id, section_id, offset]`` lists is referred to as a ``CompartmentLocation``.
-*   ``node_id``: The ID of the node within the specified ``population``.
-*   ``section_index``: The global index of a given section within its cell. NOTE: this is NOT the NEURON section index (e.g. 10 for ``dend[10]``). The ``section_index`` is calculated similar to ``get_section_index`` function of the neurodamus repository in `neurodamus.reports.py <https://github.com/openbraininstitute/neurodamus/blob/1e8b00e55bcc08e9047d6c9a48d068c463c53aef/neurodamus/report.py#L6>`_.
-*   ``offset``: The fractional distance along the section (0<= offset <=1). NOTE: offset for a section is similar to the NEURON segment location e.g. 0.5 in ``dend[10](0.5)``.
-
-
-The code preserves the order of ``node_id`` in ``compartment_set``. Such entries are acceptable:
+* ``population``: The name of the node population of the ``node_id``\s in ``compartment_set``.
+* ``compartment_set``: A list of lists, where each list is ``[node_id, section_id, offset]``. Each of the ``[node_id, section_id, offset]`` lists is referred to as a ``CompartmentLocation``.
+* ``node_id``: The ID of the node within the specified ``population``.
+* ``section_id``: The global index of a given section within its cell.
+* ``offset``: The fractional distance along the section (0<= offset <=1). NOTE: offset for a section is similar to the NEURON segment location e.g. 0.5 in ``dend[10](0.5)``.
 
-.. code-block:: json
-    
-    {
-        "compartment_set": [
-            [0, 1, 0.1],
-            [2, 5, 0.7],
-            [0, 2, 0.3]
-        ]
-    }
+.. note::
+   The ``section_id`` is NOT the NEURON section index (e.g. 10 for ``dend[10]``).
```

Comment:
you need to explain what is the `section_id` in this case. you are just saying what it is not

Reply:
it's described a few lines up

Reply:
yes, I am blind